src/test/regress/expected/multi_task_assignment_policy.out

```diff
@@ -320,7 +320,7 @@ $cmd$, 'task_assignment_test_table_2');
 SELECT count(DISTINCT value) FROM explain_outputs;
  count
 ---------------------------------------------------------------------
-     2
```

Comment:
I need to update the comments in this test.

I am planning to add some more test cases here to make sure we still use round-robin assignments when the flag is set, and locally execute the query otherwise